examples/server.rs

```diff
@@ -30,7 +32,11 @@ async fn accept(addr: String, stream: TcpStream) -> http_types::Result<()> {
     async_h1::accept(&addr, stream.clone(), |_req| async move {
         let mut res = Response::new(StatusCode::Ok);
         res.insert_header("Content-Type", "text/plain")?;
-        res.set_body("Hello");
+        let body = Body::from_reader(
+            BufReader::new(File::open("tests/fixtures/zeros.txt").await?),
```

Comment:
Was this for testing? It seems a bit too much for an example.

Reply:
Oh yeah you're right. Will revert!